Ianis/task2/ok.rb

```diff
@@ -0,0 +1,47 @@
+require "watir"
+
+class Odnoklassniki
+
+  attr_reader :browser
+  def initialize
+    @browser = Watir::Browser.new :chrome
+    browser.goto("google.com")
+    sleep(1)
+  end
+
+  def collect_links
+    search_person
+    return unless present_photo?
+    take_photo
+    { links: take_links }
+  end
+
+  def search_person
+    browser.text_field(id: "lst-ib").set("Иван Петров" + " одноклассники")
```

Comment:
searched person should not be hardcoded, move it to a variable and pass it as a method argument